stories/assets/scss/_mixins.scss

```diff
@@ -327,6 +327,12 @@
   @include transform(translateX(0));
 }
 
+%scroll_spacing {
```

Comment:
@Shubham-Sharma-77 scroll spacing doesn't seems fine to me for the class and js function and i think expand to size will be better as it is the name of animation